js/ui/drop_down_editor/ui.drop_down_editor.js

```diff
@@ -594,7 +594,8 @@ const DropDownEditor = TextBox.inherit({
             toolbarItems: this._getPopupToolbarItems(),
             onPositioned: this._popupPositionedHandler.bind(this),
             fullScreen: false,
-            contentTemplate: null
+            contentTemplate: null,
+            _wrapperClassExternal: DROP_DOWN_EDITOR_OVERLAY,
```

Comment:
what about test for this?

Reply:
You can find them here: https://github.com/DevExpress/DevExtreme/pull/22834/files/77251c7cbd76f6335cf6309d9179f06f596cbba7#diff-36f1cbaf05bec6a0c2979ea420d3869320b6dbd5657a5d32204fa9551ff8fcb7R1715